third_party/ibis/ibis_snowflake/datatypes.py

```diff
@@ -30,6 +30,10 @@ def sa_sf_numeric(_, satype, nullable=True):
         nullable=nullable,
     )
 
+@dt.dtype.register(SnowflakeDialect, BINARY)
```

Comment:
This looks great! Should we also add support for other data types that may not be added i.e DATE, TEXT, BOOLEAN, etc? 
Unless they're already supported

Reply:
Good point! I created the below table to make sure all these types listed [here on Ibis](https://github.com/ibis-project/ibis/blob/5.1.0/ibis/backends/snowflake/datatypes.py#L22) worked and and they're all supported!

```
create or replace table test_specific_data_types (
    --col_fixed fixed,
    col_real real,
    col_text text,
    col_variant variant,
    col_object object,
    col_array array,
    col_binary binary,
    col_time time,
    col_boolean boolean
);
```
![image](https://github.com/GoogleCloudPlatform/professional-services-data-validator/assets/37883833/101a8b57-4341-4cee-91b5-061dae9062a4)